dataobj/schedule_entry.h

```diff
@@ -132,8 +135,16 @@ struct schedule_entry_t
 	void set_load_before_departure(bool y) { y ? stop_flags |= LOAD_BEFORE_DEP : stop_flags &= ~LOAD_BEFORE_DEP; }
 	bool is_transfer_interval() const { return (stop_flags&TRANSFER_INTERVAL)>0; }
 	void set_transfer_interval(bool y) { y ? stop_flags |= TRANSFER_INTERVAL : stop_flags &= ~TRANSFER_INTERVAL; }
-	uint8 get_stop_flags() const { return stop_flags; }
-	void set_stop_flags(uint8 f) { stop_flags = f; }
+	uint16 get_stop_flags() const { return stop_flags; }
+	void set_stop_flags(uint8 f) { stop_flags = f+(stop_flags&0xFF00); }
```

Comment:
Why don't we just change the parameter type instead of adding `set_stop_flag2`?
```suggestion
	void set_stop_flags(uint16 f) { stop_flags = f; }
```

Reply:
We have discussed to keep the type `uint8` for now to prevent expanding the PR scope.